dspace-api/src/test/java/org/dspace/uclouvain/itemEnhancer/UCLouvainItemEnhancerTest.java

```diff
@@ -600,4 +605,212 @@ public void testConsumerMultipleConfigurationsAndMultipleEnhancement() throws Ex
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void pollerTestWithSameSourceButTwoDifferentTargets() throws Exception {
+        // Create two simple author-publication relations.
+        // The two publications are linked to the author via the name.
+        context.turnOffAuthorisationSystem();
+        Item person = ItemBuilder.createItem(context, collection)
+            .withEntityType("Person")
+            .withTitle("Pierre Kiroul")
+            .build();
+        UUID personAuthority = person.getID();
+
+        Item publication1 = ItemBuilder.createItem(context, collection)
+            .withTitle("First publication")
+            .withEntityType("Publication")
+            .withAuthor("Pierre Kiroul", personAuthority.toString())
+            .build();
+        Item publication2 = ItemBuilder.createItem(context, collection)
+        .withTitle("Second publication")
+        .withEntityType("Publication")
+        .withAuthor("Pierre Kiroul", personAuthority.toString())
+        .build();
+        context.restoreAuthSystemState();
+
+        context.commit();
+        person = context.reloadEntity(person);
+        publication1 = context.reloadEntity(publication1);
+        publication2 = context.reloadEntity(publication2);
+
+        // Update the person name
+        context.turnOffAuthorisationSystem();
+        itemService.setMetadataSingleValue(context, person, "dc", "title", null, null, "Jean Gloutitou");
+        itemService.update(context, person);
+        context.restoreAuthSystemState();
+
+        context.commit();
+        person = context.reloadEntity(person);
+
+        // We now have 2 entries to process in the database.
+        List<ItemToEnhance> itemToEnhanceEntries = uclouvainItemEnhancerService.retrieveAllItemsToUpdate(context);
+        assertThat(itemToEnhanceEntries, hasSize(2));
+
+        // Trigger the poller to update the publications.
+        uclouvainItemEnhancerUpdatePoller.checkForItemUpdate();
+
+        // Reload the entities to have the latests modifications.
+        context.commit();
+        publication1 = context.reloadEntity(publication1);
+        publication2 = context.reloadEntity(publication2);
+
+        // Check the values of the publications to make sure they have changed.
+        String publication1Author = itemService.getMetadataFirstValue(
+            publication1, "dc", "contributor", "author", null
+        );
+        assertThat(publication1Author, equalTo("Jean Gloutitou"));
+        String publication2Author = itemService.getMetadataFirstValue(
+            publication2, "dc", "contributor", "author", null
+        );
+        assertThat(publication2Author, equalTo("Jean Gloutitou"));
+
+        // Check that there are no entries remaining in the database.
+        itemToEnhanceEntries = uclouvainItemEnhancerService.retrieveAllItemsToUpdate(context);
+        assertThat(itemToEnhanceEntries, hasSize(0));
+    }
```

Comment:
Missing tests:
* Person(dc.title, person.givenName) <-> Publication (dc.contributor.author) : Create a Person with only "givenName" metadata and check that the publication has the correct value
* ... I continue to think about additional boring tests